packages/webpack.config.js

```diff
@@ -3,34 +3,50 @@ const { readdirSync } = require( 'fs' );
 
 const globalObjectKey = '__UNSTABLE__elementorPackages';
 
-const packages = readdirSync( path.resolve( __dirname, 'packages' ), { withFileTypes: true } )
-	.filter( ( dirent ) => dirent.isDirectory() )
-	.map( ( dirent ) => dirent.name )
-	.map( ( packageName ) => ( {
-		packageName,
-		packageNameCamelCase: kebabToCamelCase( packageName ),
-	} ) );
+const packages = [
+	...readdirSync( path.resolve( __dirname, 'packages' ), { withFileTypes: true } )
+		.filter( ( dirent ) => dirent.isDirectory() )
+		.map( ( dirent ) => dirent.name )
+		.map( ( name ) => ( {
+			name,
+			path: path.resolve( __dirname, `./packages/${ name }/src` ),
+		} ) ),
+	{
+		name: 'ui',
+		path: path.resolve( __dirname, `./node_modules/@elementor/ui` ),
+	},
+];
```

Comment:
maybe extract to 2 variables for "internal packages" (automatically resolved) and "3rd party" ones (hard-coded)

Reply:
Done.